### DISCUSSION TITLE
Markdown support (?)

### DISCUSSION THREAD

**Discussion by jimkohl** [LefterisXris/CodeTour#89] [Ideas]
Ran some quick checks on what is supported in Markdown (running plugin in latest version of  IntelliJ).  Not sure which type of Markdown is used for rendering, some things worked well, some not as much:

* newlines - seem to have to enter 2 newlines to get one newline between paragraphs
* links - tried [text desc](someuri.com) and while the text rendered, when I clicked went nowhere
* italics rendered, bold was hard to tell
* tables - had to add header w/ |-----|------| lines and it rendered, but no separator lines shown and text for longest first col text would but up to beginning of second column text in values

**Comment by LefterisXris** [comment]
Good points, thanks for the feedback. I can work on that and improve them. I will keep you updated
